<launch>
  <node pkg="motion_primitives"
    type="motion_primitives_graph_search"
    name="graph_search_node"
    output="screen">
    <param name="file" value="$(find motion_primitives)/motion_primitives_py/data/maps/simple.bag"/>
    <param name="topic" value="/voxel_map"/>
    </node>
</launch>